content/applications/sales/crm/performance/win_loss.rst

```diff
@@ -1,26 +1,249 @@
+3. In the resulting pop-up menu, click the current filter (by default, it is set to
+   :guilabel:`Country`) and select :guilabel:`Active` from the drop-down. The rule should now read
+   "`Active` `is` `set`".
+4. Click :guilabel:`New Rule`, and once more change the default filter to :guilabel:`Active`. Click
+   on the last part of this rule and change it to read "`Active` `is` `not set`."
+5. Click :guilabel:`Add`.
+
+.. image:: win_loss/add-custom-active-filter.png
+   :align: center
+   :alt: The Add Custom Filter menu showing two rules: (1) Active is set, and (2) Active is not set.
+
+Click off the *Search* menu to reveal the report, which now displays the total :guilabel:`Count` of
+leads, whether won or lost, grouped by their stage in the CRM pipeline. Hover over a section of the
+report to see the number of leads in that stage.
+
+.. image:: win_loss/basic-win-loss-report.png
+   :align: center
+   :alt: A basic win/loss report showing all leads whether won or lost grouped by stage.
+
+Advanced filters and groupings
+------------------------------
+
+The *Pipeline Analysis* can be customized with various filters and group by options. After
+creating a basic win/loss report, click on the :guilabel:`Search bar` and consider adding advanced
+filters and groupings to customize the report for different purposes. Some useful options
+include:
+
+- **Created On**: Adjusting this filter to a different period of time such as the last 30 days
+  or the last quarter can provide more timely results.
+- **Add Custom Filter**: Clicking :guilabel:`Add Custom Filter` and scrolling through the numerous
+  options in the drop-down opens up additional search criteria like :guilabel:`Last Stage Update` or
+  :guilabel:`Lost Reason`.
+- **Multiple Groupings**: Adding multiple *Group By* selections can split results into more
+  relevant and manageable chunks.
+
+  - Adding :guilabel:`Salesperson` or :guilabel:`Sales Team` breaks up the total count of leads in
+    each :guilabel:`Stage` by who they were assigned to.
+  - Adding :guilabel:`Medium` or :guilabel:`Source` can reveal what marketing avenues generate more
+    sales.
+- **Add Custom Group > Active**: Clicking :menuselection:`Add Custom Group --> Active` separates the
+  results in each stage into *Won (true)* or *Lost (false)*. This can show at what stage leads are
+  won and lost.
+
+.. image:: win_loss/search-panel-filters-and-group-by-options.png
+   :align: center
+   :alt: The Search menu open and the Won and Lost filters highlighted.
+
+Additional measurements and views
+---------------------------------
+
+In addition to filters and group by options, the *Pipeline Analysis* can also display different
+measurements, chart types, and :ref:`report views <reporting/views>`. Need to know something other
+than the total count of leads? Change the measurement to something more valuable. Want to see the
+report as a spreadsheet? Switch to the :guilabel:`Pivot View`.
```

Comment:
I appreciate the personality and personability of this type of phrasing, but documentation is purely meant to instruct and guide the reader along the path towards usability. We don't really need to "suggest" or encourage them to "try" different views/elements. We're kinda just meant to be as sterile as possible with documentation - that way, they can quickly get the information they need.

in essence docs are...

Title of thing you want to know about
- brief explanation of thing you want to know about, and how Odoo does it.

Next heading of section to show how to set up for thing you want to know about
- here's how to set up Odoo to do the thing you want to know about

Page of thing you want to know about
- here's how to get to the page where you can do the thing you want to know about. 
- here's what everything means on the page where you can do the thing you want to know about.
- here are all the various view and filter options on the page where you can do the thing you want to know about.

...and so on...

It's just an in-depth, super informative, step-by-step instruction manual for the reader. It's our job to make sure we've left no stone unturned with our explanations. Doing so ensures there's little to no confusion, and they can feel adequately taught throughout their documentation experience